update ember to 3.28 and bump dependencies

mostafa-sakhiri opened the pull request:
Hello,

I updated ember to 3.28 and bumped dependencies.

mostafa-sakhiri committed:
update ember to 3.28 and bump dependencies

mostafa-sakhiri commented:
Hello @skaterdav85, do you have time to check this ?

mostafa-sakhiri commented:
@Henridv @andrewpye @sovietspaceship2 any one can check this ?

andrewpye commented:
Hi @mostafa-sakhiri! The contributors you've tagged have each only contributed one small PR to this addon or are no longer active on GitHub. It looks like @iamdtang has been mostly inactive on here for a couple of months, but any decision on how to proceed here is his to make 🤷

Commit:
Prettify files

Commit:
Add .doorkeeperrc.js in node files

Commit:
Fix no-prototype-builtins eslint errors

/home/ctjhoa/concord/ember-changeset-conditional-validations/addon/validators/sometimes.js
  29:50  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
  36:23  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins

Commit:
Silence template-lint errors

mostafa-sakhiri commented:
> Hi @mostafa-sakhiri! The contributors you've tagged have each only contributed one small PR to this addon or are no longer active on GitHub. It looks like @iamdtang has been mostly inactive on here for a couple of months, but any decision on how to proceed here is his to make shrug

Okay, thank you @andrewpye
